content/millennium/r4/diagnostic/observation.md

```diff
@@ -55,7 +55,7 @@ Search for labs, vitals, and alcohol/tobacco use Observations that meet supplied
 
 _Implementation Notes_
 
-* The `comments` field may have RTF or other formatted data rather than simple text.
+* The `comments` field may have RTF or other formatted data rather than simple text. This is an issue that will be resolved in a future correction. We are also investigating markdown support.
```

Comment:
I think this note may be still too vague.

Reply:
This was suggested by Dennis, I am not involved in that conversation Seada can give more information on this.

Reply:
yes this is the note from dennis

> For DSTU2, we already state there is RTF.  We’d update the existing comment “This is an issue that will be resolved in a future correction” to instead say it’ll be resolved in R4.  For R4, we’d echo the existing comment about RTF / formatted data and state that we’re investigating markdown support.

Reply:
Updated it as discussed: https://github.com/cerner/fhir.cerner.com/pull/352/commits/56249e811fbbf88e70affc672b4f98dae6a85ef6

New output:

![Screen Shot 2020-03-04 at 12 07 38 PM](https://user-images.githubusercontent.com/42872353/75909221-439c6780-5e11-11ea-8dd1-86b5b06eccc5.png)